doc/modules/storage.md

```diff
@@ -0,0 +1,107 @@
+# storage
+<span class="weex-version">0.4</span>
+<a href="https://github.com/weexteam/article/wiki/%E6%AC%A2%E8%BF%8E%E5%8F%82%E4%B8%8EWeex%E4%B8%AD%E6%96%87%E6%96%87%E6%A1%A3%E7%BF%BB%E8%AF%91"  class="weex-translate incomplete">cn</a>
+
+## Summary
+
+`storage` is a series of apis,allowing you to for example add,modify or delete stored data items.
+
+
+## API
+
+### setItem(key, value, callback)
+
+When passed a key name and value, will add that key to the storage,
+or update that key's value if it already exists.
+
+#### Arguments
+
+* `key`*(string)*: the name of the key you want to store.
+* `value`*(object)*: the name of the value you want to store.
+* `callback`*(object)*: the callback function after executing this action.  
+
+##### Example
+
+```js
+var storage = require('@weex-module/storage');
+storage.setItem('foo','foo-value');
+//or
+storage.setItem('bar','bar-value',function(e){
```

Comment:
example code had better follow our code style guide (see the [eslint config file](https://github.com/alibaba/weex/blob/dev/.eslintrc)).